scripts/solve_network.py

```diff
@@ -85,21 +85,16 @@
 import pandas as pd
 import pypsa
 from _helpers import configure_logging, create_logger
+from linopy import merge
 from pypsa.descriptors import get_switchable_as_dense as get_as_dense
-from pypsa.linopf import (
-    define_constraints,
-    define_variables,
-    get_var,
-    ilopf,
-    join_exprs,
-    linexpr,
-    network_lopf,
-)
+from pypsa.optimization.abstract import optimize_transmission_expansion_iteratively
+from pypsa.optimization.optimize import optimize
+from vresutils.benchmark import memory_logger
 
 logger = create_logger(__name__)
 
 
-def prepare_network(n, solve_opts):
+def prepare_network(n, solve_opts, config=None):
```

Comment:
why optional?

Reply:
May you clarify please? Not sure I really get the point 🙂

Reply:
Frankly speaking, it has been just a copy-paste from PyPSA-Eur. But agree that it feels more natural to have `config` as a mandatory argument. Fixed in [the stacked PR](https://github.com/ekatef/pypsa-earth/pull/1): [this line](https://github.com/ekatef/pypsa-earth/blob/94e634c438597d8f12d11349855e50c2a9a4474f/scripts/solve_network.py#L97)